osu.Game/Screens/Play/Player.cs

```diff
@@ -472,6 +472,16 @@ protected virtual ScoreInfo CreateScore()
             else
                 score.User = api.LocalUser.Value;
 
+            try
+            {
+                var performanceCalculator = ruleset?.CreatePerformanceCalculator(Beatmap.Value, score);
+                var totalPp = performanceCalculator?.Calculate(new Dictionary<string, double>());
```

Comment:
Unnecessary?
```suggestion
                var totalPp = performanceCalculator?.Calculate();
```

Reply:
Shouldn't this be passing the score's mods instead?

Reply:
Nope this dictionary is to pass the score ratings from the calculator (aim, speed, etc...). the mods are passed to the score itself above.